_docs/developer/vm_install_using_vagrant.md

```diff
@@ -80,6 +80,10 @@ operating system.
    ```
    git clone https://github.com/Submitty/Submitty.git
    ```
+   
+   **NOTE:** If you are not currently part of the Submitty organization on Github, you may want to
+   [fork](https://help.github.com/en/github/getting-started-with-github/fork-a-repo)
+   the repo and use the git url from your fork instead.
```

Comment:
Let's just add a `, especially if you are looking to contribute` bit to the end here, just to be even more explicit.